Kernel/Process.cpp

```diff
@@ -3990,6 +3991,10 @@ int Process::sys$reboot()
     FS::lock_all();
     dbg() << "syncing mounted filesystems...";
     FS::sync();
+    if (ACPI::Parser::the().reboot_capable()) {
+        dbg() << "attempting reboot via ACPI";
+        ACPI::Parser::the().do_acpi_reboot();
```

Comment:
What is the purpose of `do_acpi_reboot()` returning a `bool` if we don't do anything with it?

Reply:
We talked about it in the IRC channel, currently now we don't do anything useful with it, but, later with a power manager, it will be useful to return a Boolean value for logging. Also, be aware that even if the parser indicates it is capable of rebooting with ACPI, we shouldn't trust that indication blindly, because an error can happen (and again, that's why logging is useful).

Reply:
OK, I could be wrong about this, so for now I'll make those calls to not return anything, but that could be changed in the future :)